### PR TITLE
Update peers.txt

### PR DESCRIPTION
**Pull request by YazzyYaz**

### CODE REVIEW

**Review by Bidon15 — APPROVED** [review]

File: mamaki/peers.txt [hunk 1]
```diff
@@ -28,4 +28,4 @@ a46bbdb81e66c950e3cdbe5ee748a2d6bdb185dd@161.97.168.77:26656,
 550ab50ad0df01408928a3479e643286a47b4fc9@46.4.213.197:26656,
 43e9da043318a4ea0141259c17fcb06ecff816af@141.94.73.39:43656,
 45d0154bea2e0bbffec343894072f5feab19d242@65.108.71.92:43656,
-d6fb487ff10d9878449beaa89007da15ec43057f@194.163.137.209
+d6fb487ff10d9878449beaa89007da15ec43057f@194.163.137.209:26656
```
Comment (liamsi) [inline mamaki/peers.txt:31]: maybe simply delete this if we do not have confidence that this is the right port?

Comment (YazzyYaz) [reply mamaki/peers.txt:31]: @liamsi Luka told me it's the correct port

Comment (YazzyYaz) [reply mamaki/peers.txt:31]: oh, he said "he believes it is"

Comment (YazzyYaz) [reply mamaki/peers.txt:31]: will delete then

Comment (YazzyYaz) [reply mamaki/peers.txt]: ok @liamsi i deleted it completely, please check

**Pull request merged**